vagrant/provision.sh

```diff
@@ -62,10 +62,11 @@ chmod +x /usr/local/bin/yq-"${YQ_VERSION}"
 ln -s /usr/local/bin/yq-"${YQ_VERSION}" /usr/local/bin/yq
 
 # Install docker
+LSB_RELEASE=$(lsb_release -cs)
 curl -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add -
 add-apt-repository \
    "deb [arch=amd64] https://download.docker.com/linux/ubuntu \
-   $(lsb_release -cs) \
+   ${LSB_RELEASE} \
```

Comment:
```suggestion
   "${LSB_RELEASE}" \
```

Reply:
Had to revert this part, as it's already quoted and shellcheck complains about it.